ci/update-gh-pages.sh

```diff
@@ -100,7 +100,10 @@ jsduck \
      --output="$DOCS_DIR/" \
      --eg-iframe=$TRAVIS_BUILD_DIR/docresources/eg-iframe.html \
      --processes=0 \
```

Comment:
This should be added to the jsduck-cfg.json as well, really helps to improve my command output?